core/governance/engine.go

```diff
@@ -1240,6 +1241,10 @@ func (e *Engine) updatedMarketFromProposal(p *proposal) (*types.Market, types.Pr
 	default:
 		return nil, types.ProposalErrorUnsupportedProduct, ErrUnsupportedProduct
 	}
+	// assign the current liquidation strategy if none was set on the update proposal
```

Comment:
how come we don't have this in new market (this is update market I think)?

Reply:
For this version, the liquidation strategy is still optional, the next PR will change the governance validation and all more, and liquidation strategy will no longer be optional then

Reply:
it will still be optional for the migration release, i.e. for proposals for new markets/update markets that will exist at the time of the migration.

Reply:
I've talked about this with David, he's leaning towards setting the liquidation strategy to some default value on migration.